german/Alkohol/Meer/meer.dm

```diff
@@ -0,0 +1 @@
+Sie stehen am Wasser und schauen aufs Meer hinaus.
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.